### ISSUE TITLE
Is whumph a library?

### ISSUE THREAD

**Issue by johnaweiss** [flovan/whumph#2]
hi, a bit confused by your description "a primer", which usually means a "tutorial"

**Comment by flovan** [comment]
I'll let the dictionary do the talking here 😉 
<img width="330" alt="Screenshot 2020-04-06 at 10 53 27" src="https://user-images.githubusercontent.com/1419500/78540749-e8390d00-77f4-11ea-888b-e58f784c2cee.png">

**Comment by johnaweiss** [comment]
ok, that's pr-**eye**-mer. I confused it with p-**rim**-er :D 
![image](https://user-images.githubusercontent.com/53209681/78593852-2685fa80-77fc-11ea-8acb-44780d8e0f7d.png)

I think your answer is "yes, it's a library like jquery", correct?

But paint primer goes _under_ the paint. Doesn't whumph goes over top of js?

thx

**Comment by flovan** [comment]
Yes JS would be the actual ground layer, if you really want to stretch the metaphor that thin 😅 

It was intended as a replacement to jQuery, providing a similar API, but leaning heavily on browser API's and not caring as much for older non-compatible browsers.

Note that this project is now 4-5 years old; I'm pretty sure there are better solutions out there by now!
